<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTransactionDialog</class>
 <widget class="QWidget" name="AddTransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="kategoriaLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>Kategoria</string>
   </property>
  </widget>
  <widget class="QComboBox" name="kategoriaBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Zakupy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wpływy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paliwo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wyjścia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Opłaty</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Raty</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="kwotaLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>Kwota</string>
   </property>
  </widget>
  <widget class="QLabel" name="opisLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>Opis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dataBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dataLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="typLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>Typ</string>
   </property>
  </widget>
  <widget class="QComboBox" name="wydatkiBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Wpływy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wydatki</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="dodajButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kwotaEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
